front/src/reducers/osrdconf/operationalStudiesConf/index.ts

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import type { OsrdConfState } from 'applications/operationalStudies/consts';
+
+import type { InfraStateType } from 'reducers/infra';
+import { buildInfraStateReducers, infraState } from 'reducers/infra';
+import { defaultCommonConf, buildCommonConfReducers } from 'reducers/osrdconf/osrdConfCommon';
+
+export type OperationalStudiesConfStateType = OsrdConfState & InfraStateType;
+
+export const operationalStudiesConfInitialState = { ...infraState, ...defaultCommonConf };
```

Comment:
Move infraState in osrdConfCommon (and maybe rename it initialInfraState :) )
```suggestion
export const operationalStudiesConfInitialState = { ...defaultCommonConf };
```

Reply:
Done